src/Symfony/Bundle/FrameworkBundle/Resources/config/routing.xml

```diff
@@ -40,7 +40,7 @@
             <argument type="service" id="file_locator" />
         </service>
 
-        <service id="routing.loader.service" class="Symfony\Component\Routing\Loader\DependencyInjection\ServiceRouterLoader">
+        <service id="routing.loader.container" class="Symfony\Component\Routing\Loader\ContainerLoader">
```

Comment:
Are we allowed to rename a public service?

Reply:
not without a deprecated alias (do we have this ?) as BC layer, or keeping a deprecated public service as BC layer (if that's OK to have 2 instances)

Reply:
Thank you. Actually I think renaming it is unnecessary so I will revert it. Using “service” here is right since it’s the implementation we use in the FWB.